### ISSUE TITLE
MODX 3.0.0 Template Menu in Quick and regular Edit mode

### ISSUE THREAD

**Issue by PeterBcreates** [modxcms/revolution#16135]
### Summary
The Template menu lists the first 20 templates in Quick Edit and Edit modes, and sporadically shows the Pagination navigation arrows.

### Step to reproduce
Create Document, Quick Create Document, Quick Edit, and Edit

### Observed behavior
Regardless of whether it displays the Pagination or not, it displays only 20 templates. I have 65. It's a mess because it's the original Cloud from 10 years ago.

QuickEdit1:
<img width="732" alt="QuickEdit1" src="https://user-images.githubusercontent.com/102907654/162757381-1ec9a1cf-da04-431c-90a1-b3d27624933c.png">

QuickEdit2:
<img width="822" alt="QuickEdit2" src="https://user-images.githubusercontent.com/102907654/162757426-0227e369-5a6e-4f8e-995f-5ce1f0fb904c.png">

RegEdit1: I don't know how this displayed the newer Templates and the Pagination nav.
<img width="915" alt="RegEdit1" src="https://user-images.githubusercontent.com/102907654/162757521-03082a52-5907-43d5-b931-6c5690c5bf12.png">

### Expected behavior
It should display all of the Templates.

### Environment
MODXcloud 3.0.0, upgraded from 2.8.3

**Comment by halftrainedharry** [comment]
It looks like the problem is, that the processor always returns a 'total' value of `0`.

---

Can you change this line
https://github.com/modxcms/revolution/blob/fe438bf3dd4a9863bc479e21945d05215141d853/core/src/Revolution/Processors/Element/Template/GetList.php#L47
to 
```
$c->sortby('Category.category');
```
and check if that solves the issue?

**Comment by PeterBcreates** [comment]
That worked for Quick Edit, and Regular Edit. It displayed all of the templates without issue.

It doesn't work for Create Document and Quick Create Document. Although I correctly entered the name of the template, it listed the first 3 Categories of Templates. There are 15 Categories.
<img width="615" alt="Create-Displays-3-Categories" src="https://user-images.githubusercontent.com/102907654/162795607-a73c6d39-8887-4c3d-b853-2e3e6c4ef80e.png">

**Comment by halftrainedharry** [comment]
Not sure if this is the best way to solve the issue, but you could try adding this function to the same file (`core/src/Revolution/Processors/Element/Template/GetList.php`).
```
public function initialize()
{
    $this->setDefaultProperties([
        'limit' => 0
    ]);

    return parent::initialize();
}
```
This should return all the templates if no limit is specified in the request.

**Comment by PeterBcreates** [comment]
Sorry for the delay in getting back to you.

That code worked, so Create and Quick Create now show all of the Templates.

Note about Create: the Templates don't display the same way they do with QuickCreate, which displays 4 "pages" to navigate through. In Create, it's not visually apparent that one can scroll.

**Comment by PeterBcreates** [comment]
Here's the snippet code:
<img width="621" alt="Translit Snippet" src="https://user-images.githubusercontent.com/102907654/164733649-651fc7c5-6397-4048-899e-f456251b4c7d.png">

And here's the output:
<img width="285" alt="Translit Result" src="https://user-images.githubusercontent.com/102907654/164733702-af46393a-15b2-4bfa-8db8-634bd5a259c7.png">

**Comment by smg6511** [comment]
Solved with #16160.
